woosmapgeofencingcore/src/main/java/com/webgeoservices/woosmapgeofencingcore/WoosmapSettingsCore.java

```diff
@@ -448,4 +458,57 @@ protected static String getGeofencingSDKVersion(){
         return BuildConfig.CORE_SDK_VERSION;
     }
 
+    protected static String getSHA1CertificateHash(Context context){
```

Comment:
This process can be cached to private static var.
This will avoid an extensive reading package for every API called.